vite.config.ts

```diff
@@ -0,0 +1,6 @@
+import { sveltekit } from '@sveltejs/kit/vite';
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+	plugins: [sveltekit()]
+});
```

Comment:
I think I'll be much better to use `:3000` port

Also a proxy would be helpful in case we'll be dealing with some sort of server.

```suggestion
import { sveltekit } from '@sveltejs/kit/vite';
import { defineConfig } from 'vite';

export default defineConfig({
	plugins: [sveltekit()],
	server: {
		port: 3000,
		proxy: {
			'api': 'http://localhost:8080/api'
		}
	}
});
```

Reply:
Sorry, there was a mistake in the code suggestion, can you re-edit it?